src/node.cc

```diff
@@ -1914,19 +1884,17 @@ static void SetGroups(const FunctionCallbackInfo<Value>& args) {
   if (rc == -1) {
     return env->ThrowErrnoException(errno, "setgroups");
   }
+
+  args.GetReturnValue().Set(0);
 }
 
 
 static void InitGroups(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args);
 
-  if (!args[0]->IsUint32() && !args[0]->IsString()) {
-    return env->ThrowTypeError("argument 1 must be a number or a string");
-  }
-
-  if (!args[1]->IsUint32() && !args[1]->IsString()) {
-    return env->ThrowTypeError("argument 2 must be a number or a string");
-  }
+  CHECK_EQ(args.Length(), 1);
```

Comment:
Should this not be 2?

Reply:
Right. That means we don't have any test for this function 😱